### PR TITLE
Update link for intro survey - add feedback to new courses

### PR DESCRIPTION
**Pull request by hollenberry**
Hey @github/services-training ! Added links to the new surveys and changed the wording a bit for transparency before click. 

It looks good on my end locally, but I'd ❤️  to 🚢  with two Approvals 😄

### CODE REVIEW

File: paths/git-trouble/fc-scenarios.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ main-content: |
 
   That innovative fix was **supposed** to fix everything! Instead your project is a garbage fire and you just want everything to go back to how it was. Have no fear, we can make that happen.  
 
+  ## We :heart: Feedback
+
+    Please take _literally less than a minute_ to let us know what you think of this course. You also have the option to opt-in for updates on future classes. [Start Survey](http://www.surveygizmo.com/s3/3288550/git-reset)!
```
Comment (crichID) [inline paths/git-trouble/fc-scenarios.md]: @hollenberry "you also have the option to opt-in" sounds redundant. Could we say: "You may also choose to opt-in"? or something along those lines?

File: _includes/activity/github-desktop/desktopsurvey.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+## We :heart: Feedback
+  Please take _literally less than a minute_ to let us know what you think of this course. You also have the option to opt-in for updates on future classes. [Start Survey](http://www.surveygizmo.com/s3/3288550/desktop)!
```
Comment (crichID) [inline _includes/activity/github-desktop/desktopsurvey.md]: > "you also have the option to opt-in" sounds redundant. Could we say: "You may also choose to opt-in"? or something along those lines?

@hollenberry same feedback here!

File: _includes/activity/github-CLI/clisurvey.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+## We :heart: Feedback
+  Please take _literally less than a minute_ to let us know what you think of this course. You also have the option to opt-in for updates on future classes. [Start Survey](http://www.surveygizmo.com/s3/3288550/cli)!
```
Comment (crichID) [inline _includes/activity/github-CLI/clisurvey.md]: > "you also have the option to opt-in" sounds redundant. Could we say: "You may also choose to opt-in"? or something along those lines?

@hollenberry same feedback here!

File: paths/intro-to-github/13-celebrate.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ main-content: |
 
   ## We :heart: Feedback
 
-  Please take a moment to let us know what you think of this on-demand training and sign up for updates when we release new classes. [Start Survey](http://www.surveygizmo.com/s3/2885641/On-Demand-Post-Survey)!
+  Please take _literally less than a minute_ to let us know what you think of this course. You also have the option to opt-in for updates on future classes. [Start Survey](http://www.surveygizmo.com/s3/3288550/intro-to-github)!
```
Comment (crichID) [inline paths/intro-to-github/13-celebrate.md]: > "you also have the option to opt-in" sounds redundant. Could we say: "You may also choose to opt-in"? or something along those lines?

@hollenberry same feedback here!

File: paths/git-trouble/fc-scenarios.md (hunk 1 above)
Comment (hollenberry) [inline paths/git-trouble/fc-scenarios.md]: Good call!

**Comment by hollenberry** [comment]
Thank you so much @crichID and @brianamarie !!

Merging now.

**Pull request merged**